functionalities-guide/cockpit/index.rst

```diff
@@ -566,14 +560,20 @@ Here the user can:
 -  activate the on-click interaction of the widget with the other ones;
 -  activate the updating of widget data due to the interaction with other widgets.
 
-When executing the cockpit in visualization mode, the user has also some more options for widgets. For all widget the user can use the icon |image197| to expand the widget to all page and use the icon |image198| to reduce it again.
+When executing the cockpit in visualization mode, the user has also some more options for widgets. For all widget the user can use the icon |image197| to expand the widget to all page and use the icon |image198| to reduce it again. There are also two new widget options: using th icon |image300| it is possible to capture the screenshot of the widget and clicking on the icon |image301| the data plotted on a chart or displayed in a table or crosstab are exported in an excel file.
```

Comment:
When executing the cockpit in visualization mode, the user has also some more options for widgets. For all widgets the user can use the icon |image197| to expand the widget to full page and use the icon |image198| to reduce it again. There are also two new widget options: using the icon |image300| it is possible to capture the screenshot of the widget and clicking on the icon |image301| the data plotted on a chart or displayed in a table or crosstab are exported in an Excel file.